BraveRewardsUI/Extensions/BraveRewardsExtensions.swift

```diff
@@ -4,6 +4,18 @@
 
 import Foundation
 import BraveRewards
+import DeviceCheck
+
+extension BraveRewards {
+  /// Whether or not Brave Rewards is available/can be enabled
+  public static var isAvailable: Bool {
+    #if DEBUG
+    return true
+    #else
+    return DCDevice.current.isSupported
```

Comment:
Nit: This could be in the DeviceCheck class we already have so that all `DCDevice` related stuff is in one place.. but entirely up to you. Not a problem to leave it here.